src/timeseriesflattener/flattener.py

```diff
@@ -120,6 +120,10 @@ def aggregate_timeseries(self, specs: Sequence[ValueSpecification]) -> Aggregate
             self.predictiontime_frame.df = self.predictiontime_frame.collect()  # type: ignore
             for spec in specs:
                 spec.value_frame.df = spec.value_frame.collect()  # type: ignore
+        else:
```

Comment:
what if one is lf and one is df? should we account for that?

Reply:
(right now we assume that people are consistent (i.e., both pred_time_frame and value_frame are the same type))

Reply:
True; seems reasonable to assume they are both the same type. We should talk about it in #549 I think.